qml/ChartsLayout.qml

```diff
@@ -230,6 +230,96 @@ Rectangle {
         }
     }
 
+    Rectangle {
+        id: emptyScreen
+        width: parent.width / 1.5
+        height: parent.height / 1.5
+        anchors.verticalCenter: parent.verticalCenter
+        anchors.horizontalCenter: parent.horizontalCenter
+        visible: statisticsChartBoxModel.count == 0
+
+        ColumnLayout{
+            anchors.fill: parent
+
+            Rectangle {
+                Layout.preferredWidth: emptyScreen.width * 3/5
+                Layout.preferredHeight: emptyScreen.height * 3/5
+                Layout.alignment: Qt.AlignCenter
+                color: "transparent"
+
+                Image {
+                    id: emptyScreenImage
+                    source: "/resources/images/graphs.svg"
+                    sourceSize.width: parent.width - 5
+                    sourceSize.height: parent.height - 5
+                    anchors.centerIn: parent
+                    opacity: 0.9
+                }
+            }
+
+            Rectangle {
+                Layout.fillWidth: true
+                Layout.preferredHeight: emptyScreen.height * 0.5/5
+                Layout.alignment: Qt.AlignCenter
+                color: "transparent"
+
+                Text {
+                    id: emptyScreenLabel
+                    width: parent.width
+                    height: parent.height
+                    anchors.verticalCenter: parent.verticalCenter
+                    anchors.horizontalCenter: parent.horizontalCenter
+                    horizontalAlignment: Text.AlignHCenter
+                    verticalAlignment: Text.AlignVCenter
+                    fontSizeMode: Text.Fit
+                    minimumPixelSize: 10
+                    font.pointSize: 20
+                    font.bold: true
+                    color: Theme.x11Grey
+                    text: mainApplicationView.monitors == 0 ? "Ups... no initilized monitors yet"
```

Comment:
```suggestion
                    text: mainApplicationView.monitors == 0 ? "Oops... no initialized monitors yet"
```

Reply:
I would not write "Oops" as nothing went wrong.

Reply:
It seems that the "oops" will stay